File: 3737/solution.py
from collections import defaultdict
class Solution:
    def countMajoritySubarrays(self, nums: List[int], target: int) -> int:
        res = 0
        for i in range(len(nums)):
            di = defaultdict(int)
            for j in range(i, len(nums)):
                di[nums[j]]+=1
                if 2*di[target]>j-i+1:
                    res+=1
        return res

File: 3737/solution.java
class Solution {
    public int countMajoritySubarrays(int[] nums, int target) {
        int counter = 0;
        int res = 0;
        for(int i=0;i<nums.length;i++){
            counter =0;
            for(int j=i;j<nums.length;j++){
                if(nums[j]==target)
                    counter++;
                if(2*counter>j-i+1)
                    res++;
            }
        }
        return res;
    }
}
